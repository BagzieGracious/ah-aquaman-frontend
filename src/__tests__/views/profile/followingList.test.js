import { shallow } from "enzyme";
import React from "react";
import FollowingList from "../../../views/profile/followingList";

describe("following List component tests", () => {
  it("should render without crashing", () => {
    const props = {
      list: [
        { image: "test.jpg", username: "truly", bio: "am cool" },
      ],
    };
    const wrapper = shallow(<FollowingList {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
