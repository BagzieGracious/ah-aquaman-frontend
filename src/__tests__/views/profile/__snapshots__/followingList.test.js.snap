// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`following List component tests should render without crashing 1`] = `ShallowWrapper {}`;
